src/index.tsx

```diff
@@ -1,2 +1,58 @@
-import React from 'react'
+import React from "react";
+import { Form } from "./components/form";
+import { ModalContainer } from "./components/modal-container";
+import { MoodSelector } from "./components/mood-selector";
+import { Mood, MOODS_DATA, MoodList } from "./moods";
+import { styled } from "./theme";
 
+export const DEFAULT_MOODS_LIST = MOODS_DATA["msFluent"];
+
+export interface FeedbackData {
+  mood: {
+    value: number;
+    total: number;
```

Comment:
this total looks interesting, what is this being used for again?

Reply:
I think it's easier for many developers to work with numbers than strings. So bad would be a 0/5 score for 5 total moods. And good would be a 5/5 score.

Reply:
can't this be derived from the 'value' key already? total becomes redundant then, no? 😄

Reply:
No, the total is the length of the mood list. It's useful when you have a callback function for the submit handler which doesnt have access to the mood list in the context

Reply:
ah gotcha. But then again, putting it inside the mood object itself doesn't look like the best.